<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfiguracionImagenesMapaDialog</class>
 <widget class="QDialog" name="ConfiguracionImagenesMapaDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>604</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1026</width>
    <height>604</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>50000</width>
    <height>50000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestor de Mapas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/iconos/images/iconowp_8Vn_icon.ico</normaloff>:/iconos/images/iconowp_8Vn_icon.ico</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pathFichero"/>
       </item>
       <item>
        <widget class="QPushButton" name="seleccionarImagenBoton">
         <property name="text">
          <string>Seleccionar imagen</string>
         </property>
         <property name="icon">
          <iconset resource="recursos.qrc">
           <normaloff>:/iconos/images/open16.png</normaloff>:/iconos/images/open16.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lat1Label">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Latitud superior izquierda:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lat1"/>
       </item>
       <item>
        <widget class="QLabel" name="lng1Label">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Longitud superior izquierda:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lng1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lat2Label">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Latitud inferior derecha:   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lat2"/>
       </item>
       <item>
        <widget class="QLabel" name="lng2Label">
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Longitud inferior derecha:    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lng2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="mapasHorizonBoton">
         <property name="text">
          <string>Buscar mapas de Horizon</string>
         </property>
         <property name="icon">
          <iconset resource="recursos.qrc">
           <normaloff>:/iconos/images/mp-plane-logo.ico</normaloff>:/iconos/images/mp-plane-logo.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addImagenBoton">
         <property name="text">
          <string>AÃ±adir Imagen</string>
         </property>
         <property name="icon">
          <iconset resource="recursos.qrc">
           <normaloff>:/iconos/images/plus16.png</normaloff>:/iconos/images/plus16.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1006</width>
        <height>433</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="botonera">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>botonera</sender>
   <signal>accepted()</signal>
   <receiver>ConfiguracionImagenesMapaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonera</sender>
   <signal>rejected()</signal>
   <receiver>ConfiguracionImagenesMapaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
